src/drivers/fluid_alsa.c

```diff
@@ -135,6 +136,9 @@ typedef struct {
   int port_count;
 } fluid_alsa_seq_driver_t;
 
+static int autoconn_inputs;
+static snd_seq_addr_t autoconn_dest;
```

Comment:
I wasn't happy as well, just it was the easiest to get to a first working draft. I had some confusion about the various types defined in fluid_alsa.c. In the revised patch, it was easy to move them into fluid_alsa_seq_driver_t. However, there seems to be still the duplicate of fluid_alsa_audio_driver_t::autoconn_inputs. I don't know how to reach the latter struct from a pointer to the former struct, otherwise I would have happily re-used it from there. Any hint ?

Reply:
fluid_alsa_audio_driver_t::autoconn_inputs was unused, whilst I always need fluid_alsa_seq_driver_t*, so I just made a further micro-fix removing the unused one.
(comments welcome)